### PR TITLE
Use ToString when value returned from GetValue is not a string type.

### PR DESCRIPTION
**Pull request by robertbeuligmann**
#406 Changes Row.GetString to mimic the MySql.Data GetString by using ToString() for non-string type column.

### CODE REVIEW

**Comment by robertbeuligmann** [comment]
Having GetString now automatically convert columns to string would is a major blocking change from the default MySql.Data implementation for the project I would like to convert to MySqlConnector.

File: src/MySqlConnector/Core/Row.cs [hunk 1]
```diff
@@ -225,7 +225,11 @@ public long GetInt64(int ordinal)
 
 		public DateTimeOffset GetDateTimeOffset(int ordinal) => new DateTimeOffset(DateTime.SpecifyKind(GetDateTime(ordinal), DateTimeKind.Utc));
 
-		public string GetString(int ordinal) => (string) GetValue(ordinal);
+		public string GetString(int ordinal)
+		{
+			var value = GetValue(ordinal);
+			return value is string stringValue ? stringValue : value.ToString();
```
[the remark this answers is not in the document]
Comment (robertbeuligmann) [reply src/MySqlConnector/Core/Row.cs]: It is the same conversion used in MySql.Data https://github.com/mysql/mysql-connector-net/blob/fcb0843b4140e648195be7ff2ef0f20dda14e45b/Source/MySql.Data/datareader.cs#L693

**Comment by bgrainger** [comment]
Opened mysql-net/AdoNetApiTest#4 for the data type conversion investigation.

**Comment by bgrainger** [comment]
Closing this since a culture-sensitive conversion of a retrieved value to a `string` is likely to be buggy, and since `GetValue(x).ToString()` is a possible workaround.

**Pull request closed without merging**